service/health/searcher/aws.go

```diff
@@ -2,12 +2,74 @@ package searcher
 
 import (
 	"context"
+
+	"github.com/giantswarm/microerror"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// AWSGreenConditions contains the values for a cluster state which are considered 'green'
+var AWSGreenConditions = map[string]struct{}{
```

Comment:
I mean this works but it looks very `python` to me :D

Reply:
I am surprised and disappointed by the lack of a generic data structure with an O(N) membership test :(